PWGJE/Tasks/bjetTreeMerger.cxx

```diff
@@ -0,0 +1,347 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file bjetTreeMerger.cxx
+/// \brief Task for merging time frames with b-jet information. Can not be used on hyperloop
```

Comment:
why do you need this task instead of using the merger tool locally?

Reply:
Removed